November Newsletter (#1819)
USRSE/usrse.github.io: _posts/newsletters/2025-11-01-newsletter.md

The committee also began preparations for the upcoming International RSE Survey and discussed finalizing our own internal US-RSE annual survey. These surveys will provide valuable insights into the RSE community and help us steer our initiatives more effectively.

-----------------

<a name="sc25"></a>
# 💎 **4. SC25: The International Conference for High Performance Computing, Networking, Storage, and Analysis**

<div class="alert alert-primary" role="alert">
  <strong>Join US-RSE at SC25!</strong>

SC25, the International Conference for High Performance Computing, Networking, Storage, and Analysis, is the premier global conference for the HPC community. This year’s event will be held in <strong>St. Louis, MO, USA</strong>, from <strong>November 16–21, 2025</strong>, with both in-person and virtual attendance options.
</div>

This year, we have representation from US-RSE in multiple capacities, including:

- **Dan Katz**, presenting \"Research Software Engineering in the Age of AI\" panel on Thursday, 20 Nov 2025, 1:30 pm - 3:00 pm, rooms 240-241-242 [Link here.](https://sc25.conference-program.com/presentation/?id=pan109&sess=sess351)

> - The **RSE-HPC workshop** at SC25 this year, on Sunday, 16 November 2025 from 9:00 am - 5:30 pm in room 267: [Link here.](https://sc25.conference-program.com/session/?sess=sess234)

Author: My concern here is unresponsiveness -- I'm more than happy to do a couple of blasts, but at this point folks know about both the HackMD and the Newsletter submission form and still I have to manually trawl the slack every month. If people aren't posting after multiple prompts, I have to assume they aren't as motivated as yourself for eg...

%}
-->

<!-- Include opportunity-box and event-box includes -->

{% include opportunity-box.html 
    title="Supercomputing and the Future of AI: Watch the Full Virtual Exchange Series On-Demand! 🤖" 
    preamble="
Our exciting webinar series for K–12 classrooms, presented in partnership with Reach the World, has officially wrapped! Over the past ten weeks, more than 840 students joined us to explore the fascinating world of supercomputing, quantum computing, and artificial intelligence.
Now, all episodes are available to watch on-demand along with companion journal articles that bring the topics to life for students, educators, and curious minds of all ages. Together with Reach the World, US-RSE created this virtual exchange to offer a behind-the-scenes look at how research software engineers and data scientists use cutting-edge computing to tackle real-world challenges.
"
    links="Learn more at [ReachTheWorld.org](https://reachtheworld.org/supercomputing-and-future-ai-0), and please reach out to [Sandra Gesing](mailto:sandra@us-rse.org) with any questions"
%}

{% include opportunity-box.html 
    title="FAIR4RS Survey" 

>     preamble="We are conducting a survey : Standards Inputs/Outputs survey  for Actionable FAIR4RS Task Force

Reviewer: ```suggestion
    preamble="The Actionable FAIR4RS Task Force is conducting a survey: "Standards Inputs/Outputs survey for Actionable FAIR4RS Task Force".
```

> We’re collecting how our community reads, writes and exchanges data from the software they develop/ maintain, with a focus on file formats, metadata standards, and APIs. Responses are anonymous unless you give contact details. Estimated time: 6–8 minutes."

Reviewer: ```suggestion
The task force is collecting information on how our community reads, writes, and exchanges data from the software they develop/ maintain, with a focus on file formats, metadata standards, and APIs. Responses are anonymous unless you give contact details. Estimated time: 6–8 minutes."
```

- **Kaiser, J., & Datta, S.** (2021). _"Probabilistic computing with p-bits."_ Applied Physics Letters, 119(15), 150503. [Read the article.](https://doi.org/10.1063/5.0067927)

- **McMahon, P. L.** (2023). _"The physics of optical computing."_ Nature Reviews Physics, 5(12), 717–734. [Read the article.](https://doi.org/10.1038/s42254-023-00645-5)

- **Stanford News** (2025), _"MingKwai prototype, the ‘origin of Chinese computing,’ finds a home at Stanford"_. [Read the post.](https://news.stanford.edu/stories/2025/05/mingkwai-chinese-typewriter-prototype-stanford-libraries)

- **Nature Reviews Physics** (2025), _"Physics needs research software engineers."_ Nature Reviews Physics, 7(7), 349–349. [Read the editorial.](https://doi.org/10.1038/s42254-025-00852-2)

- **Ramanathan, S., & Sengupta, A.** (2025). _"Introduction to Neuromorphic Computing."_ Cambridge University Press. [Pre-order the book.](https://www.cambridge.org/core/books/introduction-to-neuromorphic-computing/853942021D89F82AED182F9021E7E0F5)

- **Saul, B.** (2025). _"Great expectations: Unifying Statistical Theory and Programming."_ arXiv. [Read the pre-print.](https://doi.org/10.48550/arXiv.2510.09853)

> - **Nature Portfolio** (2025). _"Six factors affecting reproducibility in life science research and how to handle them."_ [Read the editorial.](https://www.nature.com/articles/d42473-019-00004-y)

Reviewer: I don't think this should be in the newsletter. It's not a peer-reviewed article or even an editorial, but an ad from a company.

Author: Now I'm confused because I wrote all of these by hand following the last mishap and now I can't find the original message that proposed it

The Community and Travel Funds Program is not currently accepting applications. We hope to obtain more funding in the future.
Please check back on the program page [here](https://us-rse.org/funds-and-awards/). -->

-----------------

<a name="news"></a> 
# 🗞️ **6. Community News** 

<!--Someone you want to shoutout? DO IT HERE!-->

### Annual General Meeting!

<div class="alert alert-primary" role="alert">
  <strong>Save the Date: US-RSE AGM on December 4, 2025!</strong>
</div>

The sixth US-RSE Association Annual General Meeting (AGM) will take place on Thursday, December 4th, 2025 at 1 PM ET / 10 AM PT.

Registration is free, but required for all attendees, so please make sure to [sign up here](https://us-rse.org/events/2025/2025-12-AGM/). The agenda will include:

- State of the US-RSE Association: The 2025 year in review and plans for 2026
- Updates from our Executive Director
- Updates from Working Groups
- Meet the Steering Committee Candidates
- Financial Status and Budget Review
- US-RSE leadership Q&A & Open Member Discussion

### Steering Committee Election Candidates

> The 17 candidates running for the 5 open Steering Committee seats this year have been announced! You can find their bios and statements on the [candidates page](https://us-rse.org/2025-11-11-sc-candidates/). To vote, visit the #election channel in Slack — please note that you must be a US-RSE member to participate in the election. Visit the [elections page](https://us-rse.org/about/election/) to learn more.

Author: Thanks! As you'll read, we're moving the cadence from end of month to beginning/mid of month so it'll take some getting used to

### Newsletter Cadence Update

Notice anything odd about the timing of this newsletter? We are shifting our newsletter cadence to come out at the beginning of each month, rather than the end of the month. This will help us better align with other US-RSE administration teams and provide a more consistent schedule for our readers. Additionally, if you'd like to help us review and edit future newsletters, please reach out to [Tinashe M. Tapera](mailto:tinashemtapera@gmail.com)

<a name="community-calls"></a>
### **Community Calls**

<!--Community call news goes here-->

The next community call will be on Thursday, November 13, 2025, 12pm ET/11am CT/10am MT/9am PT

**Topic: Show Me the Money (for Research Software Engineering)**

**Abstract:**

> It’s no secret—funding for RSEs and RSE work can sometimes feel like hunting for treasure without a map. In this month’s community call, we’re diving into the big questions: How do you find funding? How do you actually secure it? And perhaps the trickiest part—how do you convince funders that your project (or your role as an RSE) is worth investing in?

Reviewer 2: ```suggestion
It’s no secret — funding for RSEs and RSE work can sometimes feel like hunting for treasure without a map. In this month’s community call, we’re diving into the big questions: How do you find funding? How do you actually secure it? And perhaps the trickiest part—how do you convince funders that your project (or your role as an RSE) is worth investing in?
```

**We’re looking for volunteers!**

Help us bring the RSE voice to AGU by staffing our booth, sharing your experiences, and answering questions from attendees.
Whether you can contribute an hour or several hours, your presence makes a difference. It’s also a great chance to meet colleagues, expand your professional network, and represent the RSE community at a major scientific venue.
"
    
    links="Visit the [AGU25 website](https://www.agu.org/annual-meeting) for more information. If you’d like to volunteer, please sign up [here](https://docs.google.com/spreadsheets/d/1YGmTGw9yq3Af5iQrFY590B_jrnETiIn9RCprkdB1jiY/edit?usp=sharing).
We plan for an RSE gathering on one of the evenings. We will provide more information nearer to the event.
We look forward to seeing you at AGU25—and to showcasing the amazing impact of RSEs on science and innovation!"

> %}

Reviewer 2: Should this one be moved up since it's happening sooner (chronologically)?

Author: I don't think it's _entirely_ necessary at this stage — we wanna get this out this afternoon. I will say, to your credit, that there is a long-term plan to do this section in a data-driven manner with some liquid code that will auto sort (similar to how I populate jobs from the yaml) but I haven't gotten around to it yet.
